docs/conceptual/runtime/contracts/contracts_module.md

```diff
@@ -107,18 +107,17 @@ blockchain.
 ### Learn More
 
 - Learn about [ink!, a Rust based embedded domain specific
-  language](conceptual/runtime/contracts/ink.md) for developing smart contracts for the SRML
+  language](conceptual/runtime/contracts/ink.md) for developing smart contracts for Substrate Pallete
```

Comment:
Should be "for the contracts pallet" for consistency with the other stuff above no?

Reply:
Yes, this was missed by the find replace because of newline :/